### PR TITLE
EC0-5730 Improved chat Component 

### PR DESCRIPTION
**Pull request by Raulkg**
- Added Configurable Emoji Feature 
- Reduced Header height
- Modified Design w.r.t new figma design in the ticket 
- Added new Icons 
- Fixed landscape issue 

#### relevant ticket
[ECO-5730](https://tokbox.atlassian.net/browse/ECO-5730)

### CODE REVIEW

File: web/js/chatView.js [hunk 1]
```diff
@@ -89,8 +91,20 @@
     chatContainer = chatWndElem.querySelector('#chatMsgs');
     chatContent = chatContainer.querySelector('ul');
     chatForm = chatWndElem.querySelector('#chatForm');
+    emojiPicker = document.querySelector('emoji-picker');
+    toggleEmojiBtn = chatWndElem.querySelector('#addEmoji');
   }
 
+  const onEmojiClicked = event => chatMsgInput.value += (" "+(event.detail.unicode)+" ")
```
Comment (emilianop11) [inline web/js/chatView.js:98]: Missing semicolon and please add some spaces after and before `+` signs.

**Review by emilianop11 — CHANGES_REQUESTED** [review]
I wasnt able to test this on my real phone due to some problems with ngrok.
I tested it using devTools and it looks fantastic! Loved the emojis inclusion.
Please just take care of the little linting coments I made

File: web/js/chatView.js [hunk 2]
```diff
@@ -89,8 +91,20 @@
...
+  const toggleEmojiView = evt => {
+      evt.preventDefault();
+    if (emojiPicker.style.display == "none") {
```
Comment (emilianop11) [inline web/js/chatView.js:101]: Please compare using `===`

File: web/js/chatView.js (hunk 1 above)
Comment (Raulkg) [reply web/js/chatView.js]: Agreed and implemented changes as per suggestion

**Review by emilianop11 — APPROVED** [review]

File: web/js/chatView.js (hunk 2 above)
Comment (Raulkg) [reply web/js/chatView.js]: Agreed and implemented changes as per suggestion

**Pull request merged**